config/options.go

```diff
@@ -441,58 +441,75 @@ func (o *Options) HTTPClientUserAgent() string {
+func (o *Options) KeyValueMap() (map[string]interface{}, []string) {
+	keyValues := make(map[string]interface{})
+	keyValues["ADMIN_PASSWORD"] = o.adminPassword
+	keyValues["ADMIN_USERNAME"] = o.adminUsername
+	keyValues["AUTH_PROXY_HEADER"] = o.authProxyHeader
+	keyValues["AUTH_PROXY_USER_CREATION"] = o.authProxyUserCreation
+	keyValues["BASE_PATH"] = o.basePath
+	keyValues["BASE_URL"] = o.baseURL
+	keyValues["BATCH_SIZE"] = o.batchSize
+	keyValues["CERT_CACHE"] = o.certCache
+	keyValues["CERT_DOMAIN"] = o.certDomain
+	keyValues["CERT_FILE"] = o.certFile
+	keyValues["CLEANUP_ARCHIVE_READ_DAYS"] = o.cleanupArchiveReadDays
+	keyValues["CLEANUP_ARCHIVE_UNREAD_DAYS"] = o.cleanupArchiveUnreadDays
+	keyValues["CLEANUP_FREQUENCY_HOURS"] = o.cleanupFrequencyHours
+	keyValues["CLEANUP_REMOVE_SESSIONS_DAYS"] = o.cleanupRemoveSessionsDays
+	keyValues["CREATE_ADMIN"] = o.createAdmin
+	keyValues["DATABASE_MAX_CONNS"] = o.databaseMaxConns
+	keyValues["DATABASE_MIN_CONNS"] = o.databaseMinConns
+	keyValues["DATABASE_URL"] = o.databaseURL
+	keyValues["DEBUG"] = o.debug
+	keyValues["HSTS"] = o.hsts
+	keyValues["HTTPS"] = o.HTTPS
+	keyValues["HTTP_CLIENT_MAX_BODY_SIZE"] = o.httpClientMaxBodySize
+	keyValues["HTTP_CLIENT_PROXY"] = o.httpClientProxy
+	keyValues["HTTP_CLIENT_TIMEOUT"] = o.httpClientTimeout
+	keyValues["HTTP_CLIENT_USER_AGENT"] = o.httpClientUserAgent
+	keyValues["HTTP_SERVICE"] = o.httpService
+	keyValues["KEY_FILE"] = o.certKeyFile
+	keyValues["LISTEN_ADDR"] = o.listenAddr
+	keyValues["LOG_DATE_TIME"] = o.logDateTime
+	keyValues["MAINTENANCE_MESSAGE"] = o.maintenanceMessage
+	keyValues["MAINTENANCE_MODE"] = o.maintenanceMode
+	keyValues["METRICS_ALLOWED_NETWORKS"] = o.metricsAllowedNetworks
+	keyValues["METRICS_COLLECTOR"] = o.metricsCollector
+	keyValues["METRICS_REFRESH_INTERVAL"] = o.metricsRefreshInterval
+	keyValues["OAUTH2_CLIENT_ID"] = o.oauth2ClientID
+	keyValues["OAUTH2_CLIENT_SECRET"] = o.oauth2ClientSecret
+	keyValues["OAUTH2_OIDC_DISCOVERY_ENDPOINT"] = o.oauth2OidcDiscoveryEndpoint
+	keyValues["OAUTH2_PROVIDER"] = o.oauth2Provider
+	keyValues["OAUTH2_REDIRECT_URL"] = o.oauth2RedirectURL
+	keyValues["OAUTH2_USER_CREATION"] = o.oauth2UserCreationAllowed
+	keyValues["POCKET_CONSUMER_KEY"] = o.pocketConsumerKey
+	keyValues["POLLING_FREQUENCY"] = o.pollingFrequency
+	keyValues["POLLING_SCHEDULER"] = o.pollingScheduler
+	keyValues["PROXY_IMAGES"] = o.proxyImages
+	keyValues["ROOT_URL"] = o.rootURL
+	keyValues["RUN_MIGRATIONS"] = o.runMigrations
+	keyValues["SCHEDULER_ENTRY_FREQUENCY_MAX_INTERVAL"] = o.schedulerEntryFrequencyMaxInterval
+	keyValues["SCHEDULER_ENTRY_FREQUENCY_MIN_INTERVAL"] = o.schedulerEntryFrequencyMinInterval
+	keyValues["SCHEDULER_SERVICE"] = o.schedulerService
+	keyValues["SERVER_TIMING_HEADER"] = o.serverTimingHeader
+	keyValues["WORKER_POOL_SIZE"] = o.workerPoolSize
+
+	keys := make([]string, 0, len(keyValues))
+	for key := range keyValues {
+		keys = append(keys, key)
+	}
+	sort.Strings(keys)
+
+	return keyValues, keys
```

Comment:
You can return the options already sorted as a list instead of duplicating the same logic twice.

Here is an example:

```golang
type Option struct {
	Key   string
	Value interface{}
}

func (o *Options) SortedOptions() []*Option {
	var keyValues = map[string]interface{}{
		"ADMIN_PASSWORD":            o.adminPassword,
		"CLEANUP_ARCHIVE_READ_DAYS": o.cleanupArchiveReadDays,
                // etc....
	}

	keys := make([]string, 0, len(keyValues))
	for key := range keyValues {
		keys = append(keys, key)
	}
	sort.Strings(keys)

	var sortedOptions []*Option
	for _, key := range keys {
		sortedOptions = append(sortedOptions, &Option{Key: key, Value: keyValues[key]})
	}
	return sortedOptions
}
```